projects/lib/src/providers/google-login-provider.ts

```diff
@@ -52,13 +52,15 @@ export class GoogleLoginProvider extends BaseLoginProvider {
         let user: SocialUser = new SocialUser();
 
         const profile = this.auth2.currentUser.get().getBasicProfile();
+        const authResponse = this.auth2.currentUser.get().getAuthResponse();  // get complete authResponse object
         user.id = profile.getId();
         user.name = profile.getName();
         user.email = profile.getEmail();
         user.photoUrl = profile.getImageUrl();
         user.firstName = profile.getGivenName();
         user.lastName = profile.getFamilyName();
         user.response = profile;
```

Comment:
Change this to return `authResponse` instead. We'll treat it as backward incompatible bug fix.

Reply:
Ok I will make this change